packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+
+export default class TextField extends React.Component<{||}> {
+    render(): React.Node {
+        return (
+            <View style={[styles.container, styles.default]}>
+                <input
+                    className={css([
+                        styles.input,
+                        typographyStyles.LabelMedium,
+                    ])}
+                    placeholder="Placeholder"
+                />
+            </View>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    container: {
+        width: 288,
+        height: 40,
+        borderRadius: 4,
+        boxSizing: "border-box",
+        padding: "10px 10px 10px 16px",
+    },
+    default: {
+        background: Color.white,
+        border: `1px solid ${Color.offBlack16}`,
+        color: Color.offBlack,
+    },
+    error: {
+        background: `linear-gradient(0deg, rgba(217, 41, 22, 0.06), rgba(217, 41, 22, 0.06)), ${Color.white}`,
```

Comment:
🤔 I see that this is the value in Figma, but I think that gradient is using the same color in both steps, so maybe using a solid color instead of a gradient would get the same result.